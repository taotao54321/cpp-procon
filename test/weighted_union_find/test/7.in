11 50
1 7 8
0 7 8 2
1 3 8
1 0 2
1 6 4
1 9 8
1 2 10
0 1 7 133
0 6 9 43
0 1 6 113
1 0 1
1 6 1
0 6 8 22
0 2 3 30
0 5 8 37
1 7 8
1 3 5
1 5 10
1 0 9
0 3 10 118
1 1 10
1 3 2
1 8 6
0 4 7 57
0 8 9 21
1 5 3
0 8 10 39
1 0 7
1 7 8
1 2 5
1 9 7
0 1 10 174
0 0 6 115
0 0 3 58
0 2 3 30
0 2 3 30
0 1 9 156
0 0 9 158
0 3 7 77
0 6 10 61
0 3 9 100
0 7 8 2
0 7 9 23
0 3 10 118
1 2 1
1 6 1
0 5 8 37
0 4 9 80
0 1 4 76
0 9 10 18
