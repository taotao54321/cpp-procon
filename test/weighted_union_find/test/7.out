?
?
?
?
?
?
?
-113
2
?
?
?
?
-30
-22
?
?
2
72
-23
-26
-113
